app/models/foreman_openscap/arf_report.rb

```diff
@@ -192,5 +201,22 @@ def self.references_links(references)
       end
       html_links.join(', ')
     end
+
+    def self.check_msg_for_changes(msg, incoming_data)
```

Comment:
Based on the method name I would it expect it only verifies whether msg differs from incoming data. But in fact it updates the msg object and even saves it. I'd recommend renaming it to `update_msg_with` or something similar. In ideal world, this would be `Message` instance method. Ideally core would provide interface for updating messages... If you have free cycles I'd consider sending such PR there but having the method here is fine for now.